Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person/GetNearestDrivers.hs

```diff
@@ -41,7 +43,7 @@ getNearestDrivers ::
   Maybe Seconds ->
   m [NearestDriversResult]
 getNearestDrivers mbVariant fromLocLatLong radiusMeters merchantId onlyNotOnRide mbDriverPositionInfoExpiry = do
-  driverLocs <- Int.getDriverLocsWithCond merchantId mbDriverPositionInfoExpiry fromLocLatLong radiusMeters
+  driverLocs <- Int.getDriverLocsWithCond merchantId mbDriverPositionInfoExpiry fromLocLatLong radiusMeters mbVariant False
```

Comment:
We need not send `mbVariant` and `onRide` fields since we are anyways having this filter in the next line.
Please fix in next PR